src/style.css

```diff
@@ -0,0 +1,549 @@
+  flex-direction: column;
+  margin: 0;
+  padding: 0;
+  border-top: 1px solid rgba(0, 0, 0, 0.2);
+  position: relative;
+}
+.comment .reply {
+  position: absolute;
+  top: 4px;
+  left: -16px;
+}
+.comment .comment {
+  margin: 0;
+  padding: 8px 0 0 8px;
+  border-top: 1px solid rgba(0, 0, 0, 0.2);
+}
+.comment .comment::before {
+  width: 8px;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  content: " ";
+  display: block;
+  position: absolute;
+  padding: 0;
+  margin: 0;
+  top: 0;
+  left: 0;
+}
+.comment .details {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  color: dimgrey;
+  padding: 8px;
+}
+.comment .details .user {
+  font-weight: bold;
+  margin: 0 8px 0 0;
+}
+.comment .content {
+  padding: 0 8px 8px 8px;
+  line-height: 1.4em;
+}
+.comment .content p {
+  margin: 0;
+}
+
+.container {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  width: 100%;
+  overflow: auto;
+}
+
+.pagination {
+  padding: 1em;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+.pagination button {
+  margin: 0 0.5em;
+  padding: 0.5em;
+  border: 0;
+  background-color: transparent;
+  color: black;
+  font-size: 1em;
+  cursor: pointer;
+}
+.pagination button[disabled] {
+  opacity: 0.5;
+  cursor: auto;
+  pointer-events: none;
+}
+
+.loading {
+  pointer-events: none;
+}
+.loading .pagination, .loading article {
+  opacity: 0.3;
+  filter: blur(2px);
+}
+
+.offline-content {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  flex-direction: column;
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
+  flex: 1;
+  width: 80%;
+  height: 100vh;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  font-size: 1em;
+  margin: auto;
+}
+.offline-content .logo-icon {
+  width: 25vw;
+  height: 25vh;
+  background-size: 100% 100%;
+  opacity: 0.3;
+}
+
+.top-view {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  height: inherit;
+}
+
+@keyframes donut-spin {
+  0% {
+    transform: translateX(-50%) rotate(0deg);
+  }
+  100% {
+    transform: translateX(-50%) rotate(360deg);
+  }
+}
+.donut {
+  display: block;
+  border: 0.75em solid dimgray;
+  border-left-color: #ffb400;
+  border-radius: 50%;
+  width: 2em;
+  height: 2em;
+  padding: 2em;
+  animation: donut-spin 1s linear infinite;
+}
+
+.pagination + .donut {
+  display: none;
+}
+
+.loading .donut {
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 1;
+}
+
+.dark {
```

Comment:
I was thinking we should use `@media (prefers-color-scheme: dark)` here so we could have dark mode for users who already have opted in for it, system wide.